src/c++/perf_analyzer/genai-pa/genai_pa/llm_inputs/llm_inputs.py

```diff
@@ -37,54 +55,105 @@ class LlmInputs:
     DEFAULT_LENGTH = 100
     MINIMUM_LENGTH = 1
 
+    EMPTY_JSON_IN_VLLM_PA_FORMAT = {"data": []}
+    EMPTY_JSON_IN_TRTLLM_PA_FORMAT = {"data": []}
     EMPTY_JSON_IN_OPENAI_PA_FORMAT = {"data": [{"payload": []}]}
 
+    dataset_url_map = {OPEN_ORCA: OPEN_ORCA_URL, CNN_DAILY_MAIL: CNN_DAILYMAIL_URL}
+
     @classmethod
-    def create_openai_llm_inputs(
+    def create_llm_inputs(
         cls,
-        url: str = OPEN_ORCA_URL,
+        input_type: InputType,
+        input_format: InputFormat,
+        output_format: OutputFormat,
+        model_name: str = "",
+        input_filename: str = "",
         starting_index: int = DEFAULT_STARTING_INDEX,
         length: int = DEFAULT_LENGTH,
-        model_name: str = None,
+        add_model_name: bool = False,
```

Comment:
Can this be derived from the output format type?

Reply:
discussed offline.
potentially derivable from output model type but leaving as is for now to enable flexibility.